### ISSUE TITLE
A few tweaks?

### ISSUE THREAD

**Issue by thebaldgeek** [hotNipi/node-red-contrib-ui-value-trail#2]
I would like to start out by saying how much I appreciate this node and the help you offer in the Node-RED forums.
I am a huge sparkline fan and so use this node on a lot of my dashboard screens.

The tweaks.
1.When switching between dashboard screens I have found that the values are missing more often than not. Is it possible to get them to update every time a new value is received, even if its the same value. That way they would be redrawn more often and would show up quicker.
On the flip side, when the values are zero more often than not I get huge (12+ digits) negative numbers shown.

2. The number of data points is a little odd. A max of 24 is not divisible by anything in Node-RED that I can see. Is it possible to go to a selectable max of 30 or better still 60. (See tweak 3).

3. I know the scale is timeless, but 45 seconds is a little odd for Node-RED where so much is done in seconds or minutes. I mostly use your node to show msg.payload rates over time, so use per second or 1 minute rates. (From the msg-speed node).
Better still would be a simple check box option where we could have a 30 second or 60 second time scale and set the number of data points to suit.

Thanks again.

**Comment by hotNipi** [comment]
Hi. 
The main reason of those limitations is that I made this node to be as lightweight as possible. 
And the outcome should not tell more than it does today. Giving first impressions of measured process. I did the node to make all this possible without usage of line chart, which is heavy to render in most cases for devices with low computing power.   
If I make it to accept time frame then it will be basically line chart. But the line chart already exists so ... no.

The number of data points - this can be changed. There should be some kind of "wise usage notification", which should tell that it may harm performance if too many points needed to rendered. 
But - 
24 points per unit
By default the unit is 48px
2 points will have distance 2px
line width minimum 1px 
To squeeze it more, the line will be lost and you'll see filled area.

(12+ digits) negative numbers - that's a bug i also noticed lately. Will be fixed.
